debian/changelog

```diff
@@ -0,0 +1,5 @@
+key-mapper (0.8.2) UNRELEASED; urgency=low
```

Comment:
This shouldn’t be a “native” package, so:

```suggestion
key-mapper (0.8.2-1) UNRELEASED; urgency=low
```

Reply:
Yes. Like I said, we should switch to quilt.

What's your opinion on to the hosting? github or salsa? And which tool is your preferred one? `git-buildpackage`?

Can you provide the patch?

Reply:
I pushed my packaging tree to https://salsa.debian.org/python-team/packages/key-mapper if you want to take a look. I prefer `gbp`, yes, and would follow the Python team guidelines for this project.